### PR TITLE
media-sound/forked-daapd:  Fix --disable-*

### LINKED ISSUE #18: =media-sound/forked-daapd: configure: error: Package requirements ( flac ) were not met:
The following makes me wonder if flac is truly optional.

Let me know what you think, @erikmack.
```
=media-sound/forked-daapd-23.2[]
emerge -q --backtrack=130 =media-sound/forked-daapd-23.2
------------------------------------------------------------------------

 * IMPORTANT: 12 news items need reading for repository 'gentoo'.
 * Use eselect news read to view new items.

>>> Verifying ebuild manifests
>>> Running pre-merge checks for dev-java/icedtea-bin-7.2.5.5
>>> Emerging (1 of 113) media-fonts/dejavu-2.34::gentoo
>>> Installing (1 of 113) media-fonts/dejavu-2.34::gentoo
>>> Emerging (2 of 113) dev-libs/libunistring-0.9.5::gentoo
>>> Installing (2 of 113) dev-libs/libunistring-0.9.5::gentoo
>>> Emerging (3 of 113) dev-libs/libdaemon-0.14-r2::gentoo
>>> Installing (3 of 113) dev-libs/libdaemon-0.14-r2::gentoo
>>> Emerging (4 of 113) media-libs/giflib-4.1.6-r3::gentoo
>>> Installing (4 of 113) media-libs/giflib-4.1.6-r3::gentoo
>>> Emerging (5 of 113) app-text/poppler-data-0.4.7::gentoo
>>> Installing (5 of 113) app-text/poppler-data-0.4.7::gentoo
>>> Emerging (6 of 113) dev-libs/lzo-2.08::gentoo
>>> Installing (6 of 113) dev-libs/lzo-2.08::gentoo
>>> Emerging (7 of 113) media-libs/libpng-1.6.16::gentoo
>>> Installing (7 of 113) media-libs/libpng-1.6.16::gentoo
>>> Emerging (8 of 113) media-libs/lcms-2.6-r1::gentoo
>>> Installing (8 of 113) media-libs/lcms-2.6-r1::gentoo
>>> Emerging (9 of 113) dev-libs/gobject-introspection-common-1.42.0::gentoo
>>> Installing (9 of 113) dev-libs/gobject-introspection-common-1.42.0::gentoo
>>> Emerging (10 of 113) media-libs/tiff-4.0.3-r6::gentoo
>>> Installing (10 of 113) media-libs/tiff-4.0.3-r6::gentoo
>>> Emerging (11 of 113) dev-libs/vala-common-0.26.2::gentoo
>>> Installing (11 of 113) dev-libs/vala-common-0.26.2::gentoo
>>> Emerging (12 of 113) virtual/ttf-fonts-1::gentoo
>>> Installing (12 of 113) virtual/ttf-fonts-1::gentoo
>>> Emerging (13 of 113) dev-java/java-config-wrapper-0.16::gentoo
>>> Installing (13 of 113) dev-java/java-config-wrapper-0.16::gentoo
>>> Emerging (14 of 113) sys-devel/bc-1.06.95-r1::gentoo
>>> Installing (14 of 113) sys-devel/bc-1.06.95-r1::gentoo
>>> Emerging (15 of 113) sys-apps/baselayout-java-0.1.0::gentoo
>>> Installing (15 of 113) sys-apps/baselayout-java-0.1.0::gentoo
>>> Emerging (16 of 113) net-dns/libidn-1.30::gentoo
>>> Installing (16 of 113) net-dns/libidn-1.30::gentoo
>>> Emerging (17 of 113) app-eselect/eselect-java-0.1.0::gentoo
>>> Installing (17 of 113) app-eselect/eselect-java-0.1.0::gentoo
>>> Emerging (18 of 113) app-text/docbook-xml-dtd-4.4-r2::gentoo
>>> Installing (18 of 113) app-text/docbook-xml-dtd-4.4-r2::gentoo
>>> Emerging (19 of 113) app-text/docbook-xml-dtd-4.2-r2::gentoo
>>> Installing (19 of 113) app-text/docbook-xml-dtd-4.2-r2::gentoo
>>> Emerging (20 of 113) app-text/xmlto-0.0.26::gentoo
>>> Installing (20 of 113) app-text/xmlto-0.0.26::gentoo
>>> Emerging (21 of 113) app-text/qpdf-5.1.1::gentoo
>>> Installing (21 of 113) app-text/qpdf-5.1.1::gentoo
>>> Emerging (22 of 113) dev-lang/nasm-2.11.08::gentoo
>>> Installing (22 of 113) dev-lang/nasm-2.11.08::gentoo
>>> Emerging (23 of 113) sys-devel/automake-1.15::gentoo
>>> Installing (23 of 113) sys-devel/automake-1.15::gentoo
>>> Emerging (24 of 113) sys-devel/automake-1.11.6-r1::gentoo
>>> Installing (24 of 113) sys-devel/automake-1.11.6-r1::gentoo
>>> Emerging (25 of 113) dev-libs/libtasn1-4.5::gentoo
>>> Installing (25 of 113) dev-libs/libtasn1-4.5::gentoo
>>> Emerging (26 of 113) media-libs/alsa-lib-1.0.29::gentoo
>>> Installing (26 of 113) media-libs/alsa-lib-1.0.29::gentoo
>>> Emerging (27 of 113) app-text/libpaper-1.1.24-r2::gentoo
>>> Installing (27 of 113) app-text/libpaper-1.1.24-r2::gentoo
>>> Emerging (28 of 113) media-libs/libjpeg-turbo-1.3.1::gentoo
>>> Installing (28 of 113) media-libs/libjpeg-turbo-1.3.1::gentoo
>>> Emerging (29 of 113) dev-libs/antlr-c-3.5::gentoo
>>> Installing (29 of 113) dev-libs/antlr-c-3.5::gentoo
>>> Emerging (30 of 113) dev-db/sqlite-3.8.10.2::gentoo
>>> Installing (30 of 113) dev-db/sqlite-3.8.10.2::gentoo
>>> Emerging (31 of 113) dev-util/patchelf-0.8::gentoo
>>> Installing (31 of 113) dev-util/patchelf-0.8::gentoo
>>> Emerging (32 of 113) media-libs/jbig2dec-0.11-r1::gentoo
>>> Installing (32 of 113) media-libs/jbig2dec-0.11-r1::gentoo
>>> Emerging (33 of 113) app-arch/cpio-2.11-r3::gentoo
>>> Installing (33 of 113) app-arch/cpio-2.11-r3::gentoo
>>> Emerging (34 of 113) dev-libs/nettle-2.7.1-r4::gentoo
>>> Installing (34 of 113) dev-libs/nettle-2.7.1-r4::gentoo
>>> Emerging (35 of 113) dev-scheme/guile-1.8.8-r1::gentoo
>>> Installing (35 of 113) dev-scheme/guile-1.8.8-r1::gentoo
>>> Emerging (36 of 113) dev-lang/yasm-1.2.0-r1::gentoo
>>> Installing (36 of 113) dev-lang/yasm-1.2.0-r1::gentoo
>>> Emerging (37 of 113) virtual/jpeg-0-r2::gentoo
>>> Installing (37 of 113) virtual/jpeg-0-r2::gentoo
>>> Emerging (38 of 113) app-arch/rpm2targz-9.0.0.5g::gentoo
>>> Installing (38 of 113) app-arch/rpm2targz-9.0.0.5g::gentoo
>>> Emerging (39 of 113) sys-devel/autogen-5.18.1::gentoo
>>> Installing (39 of 113) sys-devel/autogen-5.18.1::gentoo
>>> Emerging (40 of 113) virtual/jpeg-62::gentoo
>>> Installing (40 of 113) virtual/jpeg-62::gentoo
>>> Emerging (41 of 113) media-video/ffmpeg-2.6.3::gentoo
>>> Installing (41 of 113) media-video/ffmpeg-2.6.3::gentoo
>>> Emerging (42 of 113) media-fonts/urw-fonts-2.4.9::gentoo
>>> Installing (42 of 113) media-fonts/urw-fonts-2.4.9::gentoo
>>> Emerging (43 of 113) virtual/ffmpeg-9-r2::gentoo
>>> Installing (43 of 113) virtual/ffmpeg-9-r2::gentoo
>>> Emerging (44 of 113) x11-proto/xproto-7.0.27::gentoo
>>> Installing (44 of 113) x11-proto/xproto-7.0.27::gentoo
>>> Emerging (45 of 113) media-libs/freetype-2.5.5::gentoo
>>> Installing (45 of 113) media-libs/freetype-2.5.5::gentoo
>>> Emerging (46 of 113) x11-proto/xextproto-7.3.0::gentoo
>>> Installing (46 of 113) x11-proto/xextproto-7.3.0::gentoo
>>> Emerging (47 of 113) x11-proto/inputproto-2.3.1::gentoo
>>> Installing (47 of 113) x11-proto/inputproto-2.3.1::gentoo
>>> Emerging (48 of 113) x11-proto/renderproto-0.11.1-r1::gentoo
>>> Installing (48 of 113) x11-proto/renderproto-0.11.1-r1::gentoo
>>> Emerging (49 of 113) media-fonts/font-util-1.3.0::gentoo
>>> Installing (49 of 113) media-fonts/font-util-1.3.0::gentoo
>>> Emerging (50 of 113) x11-misc/util-macros-1.19.0::gentoo
>>> Installing (50 of 113) x11-misc/util-macros-1.19.0::gentoo
>>> Emerging (51 of 113) x11-proto/damageproto-1.2.1-r1::gentoo
>>> Installing (51 of 113) x11-proto/damageproto-1.2.1-r1::gentoo
>>> Emerging (52 of 113) dev-libs/mini-xml-2.8::gentoo
>>> Installing (52 of 113) dev-libs/mini-xml-2.8::gentoo
>>> Emerging (53 of 113) dev-libs/confuse-2.7::gentoo
>>> Installing (53 of 113) dev-libs/confuse-2.7::gentoo
>>> Emerging (54 of 113) sys-apps/dbus-1.8.16::gentoo
>>> Installing (54 of 113) sys-apps/dbus-1.8.16::gentoo
>>> Emerging (55 of 113) net-libs/gnutls-3.3.15::gentoo
>>> Installing (55 of 113) net-libs/gnutls-3.3.15::gentoo
>>> Emerging (56 of 113) x11-proto/kbproto-1.0.6-r1::gentoo
>>> Installing (56 of 113) x11-proto/kbproto-1.0.6-r1::gentoo
>>> Emerging (57 of 113) x11-proto/xf86bigfontproto-1.2.0-r1::gentoo
>>> Installing (57 of 113) x11-proto/xf86bigfontproto-1.2.0-r1::gentoo
>>> Emerging (58 of 113) x11-libs/xtrans-1.3.5::gentoo
>>> Installing (58 of 113) x11-libs/xtrans-1.3.5::gentoo
>>> Emerging (59 of 113) dev-libs/libpthread-stubs-0.3-r1::gentoo
>>> Installing (59 of 113) dev-libs/libpthread-stubs-0.3-r1::gentoo
>>> Emerging (60 of 113) x11-proto/recordproto-1.14.2-r1::gentoo
>>> Installing (60 of 113) x11-proto/recordproto-1.14.2-r1::gentoo
>>> Emerging (61 of 113) x11-proto/randrproto-1.4.0-r1::gentoo
>>> Installing (61 of 113) x11-proto/randrproto-1.4.0-r1::gentoo
>>> Emerging (62 of 113) x11-proto/compositeproto-0.4.2-r1::gentoo
>>> Installing (62 of 113) x11-proto/compositeproto-0.4.2-r1::gentoo
>>> Emerging (63 of 113) x11-libs/pixman-0.32.6::gentoo
>>> Installing (63 of 113) x11-libs/pixman-0.32.6::gentoo
>>> Emerging (64 of 113) x11-proto/fixesproto-5.0-r1::gentoo
>>> Installing (64 of 113) x11-proto/fixesproto-5.0-r1::gentoo
>>> Emerging (65 of 113) x11-libs/libXau-1.0.8::gentoo
>>> Installing (65 of 113) x11-libs/libXau-1.0.8::gentoo
>>> Emerging (66 of 113) x11-libs/libXdmcp-1.1.1-r1::gentoo
>>> Installing (66 of 113) x11-libs/libXdmcp-1.1.1-r1::gentoo
>>> Emerging (67 of 113) dev-libs/libcroco-0.6.8-r1::gentoo
>>> Installing (67 of 113) dev-libs/libcroco-0.6.8-r1::gentoo
>>> Emerging (68 of 113) media-libs/fontconfig-2.11.1-r2::gentoo
>>> Installing (68 of 113) media-libs/fontconfig-2.11.1-r2::gentoo
>>> Emerging (69 of 113) app-eselect/eselect-fontconfig-1.1::gentoo
>>> Installing (69 of 113) app-eselect/eselect-fontconfig-1.1::gentoo
>>> Emerging (70 of 113) dev-libs/libevent-2.0.22::gentoo
>>> Installing (70 of 113) dev-libs/libevent-2.0.22::gentoo
>>> Emerging (71 of 113) net-misc/curl-7.43.0::gentoo
>>> Installing (71 of 113) net-misc/curl-7.43.0::gentoo
>>> Emerging (72 of 113) app-arch/libarchive-3.1.2-r1::gentoo
>>> Installing (72 of 113) app-arch/libarchive-3.1.2-r1::gentoo
>>> Emerging (73 of 113) dev-util/cmake-3.2.2::gentoo
>>> Installing (73 of 113) dev-util/cmake-3.2.2::gentoo
>>> Emerging (74 of 113) app-text/poppler-0.32.0::gentoo
>>> Installing (74 of 113) app-text/poppler-0.32.0::gentoo
>>> Emerging (75 of 113) media-gfx/graphite2-1.2.4-r1::gentoo
>>> Installing (75 of 113) media-gfx/graphite2-1.2.4-r1::gentoo
>>> Emerging (76 of 113) dev-libs/gobject-introspection-1.42.0-r1::gentoo
>>> Installing (76 of 113) dev-libs/gobject-introspection-1.42.0-r1::gentoo
>>> Emerging (77 of 113) dev-java/javatoolkit-0.3.0-r9::gentoo
>>> Installing (77 of 113) dev-java/javatoolkit-0.3.0-r9::gentoo
>>> Emerging (78 of 113) dev-libs/atk-2.14.0::gentoo
>>> Installing (78 of 113) dev-libs/atk-2.14.0::gentoo
>>> Emerging (79 of 113) net-dns/avahi-0.6.31-r6::gentoo
>>> Installing (79 of 113) net-dns/avahi-0.6.31-r6::gentoo
>>> Emerging (80 of 113) gnome-base/gsettings-desktop-schemas-3.14.2::gentoo
>>> Installing (80 of 113) gnome-base/gsettings-desktop-schemas-3.14.2::gentoo
>>> Emerging (81 of 113) x11-proto/xcb-proto-1.11::gentoo
>>> Installing (81 of 113) x11-proto/xcb-proto-1.11::gentoo
>>> Emerging (82 of 113) dev-java/java-config-2.2.0::gentoo
>>> Installing (82 of 113) dev-java/java-config-2.2.0::gentoo
>>> Emerging (83 of 113) x11-libs/libxcb-1.11-r1::gentoo
>>> Installing (83 of 113) x11-libs/libxcb-1.11-r1::gentoo
>>> Emerging (84 of 113) x11-libs/libX11-1.6.2::gentoo
>>> Installing (84 of 113) x11-libs/libX11-1.6.2::gentoo
>>> Emerging (85 of 113) x11-libs/libXext-1.3.3::gentoo
>>> Installing (85 of 113) x11-libs/libXext-1.3.3::gentoo
>>> Emerging (86 of 113) x11-libs/libXrender-0.9.8::gentoo
>>> Installing (86 of 113) x11-libs/libXrender-0.9.8::gentoo
>>> Emerging (87 of 113) x11-libs/libXfixes-5.0.1::gentoo
>>> Installing (87 of 113) x11-libs/libXfixes-5.0.1::gentoo
>>> Emerging (88 of 113) x11-libs/gdk-pixbuf-2.30.8::gentoo
>>> Installing (88 of 113) x11-libs/gdk-pixbuf-2.30.8::gentoo
>>> Emerging (89 of 113) x11-libs/cairo-1.14.2::gentoo
>>> Installing (89 of 113) x11-libs/cairo-1.14.2::gentoo
>>> Emerging (90 of 113) x11-libs/libXi-1.7.4::gentoo
>>> Installing (90 of 113) x11-libs/libXi-1.7.4::gentoo
>>> Emerging (91 of 113) x11-libs/libXcursor-1.1.14::gentoo
>>> Installing (91 of 113) x11-libs/libXcursor-1.1.14::gentoo
>>> Emerging (92 of 113) x11-libs/libXdamage-1.1.4-r1::gentoo
>>> Installing (92 of 113) x11-libs/libXdamage-1.1.4-r1::gentoo
>>> Emerging (93 of 113) x11-libs/libXcomposite-0.4.4-r1::gentoo
>>> Installing (93 of 113) x11-libs/libXcomposite-0.4.4-r1::gentoo
>>> Emerging (94 of 113) x11-libs/libXrandr-1.4.2::gentoo
>>> Installing (94 of 113) x11-libs/libXrandr-1.4.2::gentoo
>>> Emerging (95 of 113) dev-util/gtk-update-icon-cache-3.14.13::gentoo
>>> Installing (95 of 113) dev-util/gtk-update-icon-cache-3.14.13::gentoo
>>> Emerging (96 of 113) x11-libs/libXtst-1.2.2::gentoo
>>> Installing (96 of 113) x11-libs/libXtst-1.2.2::gentoo
>>> Emerging (97 of 113) media-libs/harfbuzz-0.9.38::gentoo
>>> Installing (97 of 113) media-libs/harfbuzz-0.9.38::gentoo
>>> Emerging (98 of 113) x11-libs/pango-1.36.8::gentoo
>>> Installing (98 of 113) x11-libs/pango-1.36.8::gentoo
>>> Emerging (99 of 113) gnome-base/librsvg-2.40.9::gentoo
>>> Installing (99 of 113) gnome-base/librsvg-2.40.9::gentoo
>>> Emerging (100 of 113) x11-libs/gtk+-2.24.28-r1::gentoo
>>> Installing (100 of 113) x11-libs/gtk+-2.24.28-r1::gentoo
>>> Emerging (101 of 113) net-print/cups-2.0.3::gentoo
>>> Installing (101 of 113) net-print/cups-2.0.3::gentoo
>>> Emerging (102 of 113) app-text/ghostscript-gpl-9.15-r1::gentoo
>>> Installing (102 of 113) app-text/ghostscript-gpl-9.15-r1::gentoo
>>> Emerging (103 of 113) net-print/cups-filters-1.0.71::gentoo
>>> Installing (103 of 113) net-print/cups-filters-1.0.71::gentoo
>>> Emerging (104 of 113) dev-java/icedtea-bin-7.2.5.5::gentoo
>>> Installing (104 of 113) dev-java/icedtea-bin-7.2.5.5::gentoo
>>> Emerging (105 of 113) virtual/jdk-1.7.0::gentoo
>>> Installing (105 of 113) virtual/jdk-1.7.0::gentoo
>>> Emerging (106 of 113) virtual/jre-1.7.0::gentoo
>>> Installing (106 of 113) virtual/jre-1.7.0::gentoo
>>> Emerging (107 of 113) dev-java/ant-core-1.9.2::gentoo
>>> Installing (107 of 113) dev-java/ant-core-1.9.2::gentoo
>>> Emerging (108 of 113) dev-java/antlr-2.7.7-r5::gentoo
>>> Installing (108 of 113) dev-java/antlr-2.7.7-r5::gentoo
>>> Emerging (109 of 113) dev-java/ant-antlr-1.9.2::gentoo
>>> Installing (109 of 113) dev-java/ant-antlr-1.9.2::gentoo
>>> Emerging (110 of 113) dev-java/junit-3.8.2-r1::gentoo
>>> Installing (110 of 113) dev-java/junit-3.8.2-r1::gentoo
>>> Emerging (111 of 113) dev-java/stringtemplate-3.2::gentoo
>>> Installing (111 of 113) dev-java/stringtemplate-3.2::gentoo
>>> Emerging (112 of 113) dev-java/antlr-3.1.3-r3::gentoo
>>> Installing (112 of 113) dev-java/antlr-3.1.3-r3::gentoo
>>> Emerging (113 of 113) media-sound/forked-daapd-23.2::alunduil
>>> Failed to emerge media-sound/forked-daapd-23.2, Log file:
>>>  '/tmp/etest.logs.d/media-sound:forked-daapd-23.2:20150720-001435.log'
 * Package:    media-sound/forked-daapd-23.2
 * Repository: alunduil
 * Maintainer: erikmack@gmail.com alunduil@gentoo.org
 * USE:        abi_x86_64 amd64 elibc_glibc kernel_linux userland_GNU
 * FEATURES:   preserve-libs sandbox userpriv usersandbox
 * Adding user 'daapd' to your system ...
 *  - Userid: 103
 *  - Shell: /sbin/nologin
 *  - Home: /dev/null
 *  - Groups: (none)
 *  - GECOS: added by portage for forked-daapd
 * Running eautoreconf in '/var/tmp/portage/media-sound/forked-daapd-23.2/work/forked-daapd-23.2' ...
 * Running libtoolize --install --copy --force --automake ...
 [ ok ]
 * Running aclocal -I m4 ...
 [ ok ]
 * Running autoconf --force ...
 [ ok ]
 * Running autoheader ...
 [ ok ]
 * Running automake --add-missing --copy --foreign --force-missing ...
 [ ok ]
 * Running elibtoolize in: forked-daapd-23.2/
 *   Applying portage/1.2.0 patch ...
 *   Applying sed/1.5.6 patch ...
checking for a BSD-compatible install... /usr/bin/install -c
checking whether build environment is sane... yes
checking for a thread-safe mkdir -p... /bin/mkdir -p
checking for gawk... gawk
checking whether make sets $(MAKE)... yes
checking whether make supports nested variables... yes
checking for style of include used by make... GNU
checking for x86_64-pc-linux-gnu-gcc... x86_64-pc-linux-gnu-gcc
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables... 
checking whether we are cross compiling... no
checking for suffix of object files... o
checking whether we are using the GNU C compiler... yes
checking whether x86_64-pc-linux-gnu-gcc accepts -g... yes
checking for x86_64-pc-linux-gnu-gcc option to accept ISO C89... none needed
checking whether x86_64-pc-linux-gnu-gcc understands -c and -o together... yes
checking dependency style of x86_64-pc-linux-gnu-gcc... none
checking how to run the C preprocessor... x86_64-pc-linux-gnu-gcc -E
checking for grep that handles long lines and -e... /bin/grep
checking for egrep... /bin/grep -E
checking for ANSI C header files... yes
checking for sys/types.h... yes
checking for sys/stat.h... yes
checking for stdlib.h... yes
checking for string.h... yes
checking for memory.h... yes
checking for strings.h... yes
checking for inttypes.h... yes
checking for stdint.h... yes
checking for unistd.h... yes
checking minix/config.h usability... no
checking minix/config.h presence... no
checking for minix/config.h... no
checking whether it is safe to define __EXTENSIONS__... yes
checking for x86_64-pc-linux-gnu-gcc... (cached) x86_64-pc-linux-gnu-gcc
checking whether we are using the GNU C compiler... (cached) yes
checking whether x86_64-pc-linux-gnu-gcc accepts -g... (cached) yes
checking for x86_64-pc-linux-gnu-gcc option to accept ISO C89... (cached) none needed
checking whether x86_64-pc-linux-gnu-gcc understands -c and -o together... (cached) yes
checking dependency style of x86_64-pc-linux-gnu-gcc... (cached) none
checking build system type... x86_64-pc-linux-gnu
checking host system type... x86_64-pc-linux-gnu
checking how to print strings... printf
checking for a sed that does not truncate output... /bin/sed
checking for fgrep... /bin/grep -F
checking for ld used by x86_64-pc-linux-gnu-gcc... /usr/x86_64-pc-linux-gnu/bin/ld
checking if the linker (/usr/x86_64-pc-linux-gnu/bin/ld) is GNU ld... yes
checking for BSD- or MS-compatible name lister (nm)... /usr/bin/x86_64-pc-linux-gnu-nm -B
checking the name lister (/usr/bin/x86_64-pc-linux-gnu-nm -B) interface... BSD nm
checking whether ln -s works... yes
checking the maximum length of command line arguments... 1572864
checking how to convert x86_64-pc-linux-gnu file names to x86_64-pc-linux-gnu format... func_convert_file_noop
checking how to convert x86_64-pc-linux-gnu file names to toolchain format... func_convert_file_noop
checking for /usr/x86_64-pc-linux-gnu/bin/ld option to reload object files... -r
checking for x86_64-pc-linux-gnu-objdump... x86_64-pc-linux-gnu-objdump
checking how to recognize dependent libraries... pass_all
checking for x86_64-pc-linux-gnu-dlltool... no
checking for dlltool... no
checking how to associate runtime and link libraries... printf %s\n
checking for x86_64-pc-linux-gnu-ar... x86_64-pc-linux-gnu-ar
checking for archiver @FILE support... @
checking for x86_64-pc-linux-gnu-strip... x86_64-pc-linux-gnu-strip
checking for x86_64-pc-linux-gnu-ranlib... x86_64-pc-linux-gnu-ranlib
checking command to parse /usr/bin/x86_64-pc-linux-gnu-nm -B output from x86_64-pc-linux-gnu-gcc object... ok
checking for sysroot... no
checking for a working dd... /bin/dd
checking how to truncate binary pipes... /bin/dd bs=4096 count=1
checking for x86_64-pc-linux-gnu-mt... no
checking for mt... no
checking if : is a manifest tool... no
checking for dlfcn.h... yes
checking for objdir... .libs
checking if x86_64-pc-linux-gnu-gcc supports -fno-rtti -fno-exceptions... no
checking for x86_64-pc-linux-gnu-gcc option to produce PIC... -fPIC -DPIC
checking if x86_64-pc-linux-gnu-gcc PIC flag -fPIC -DPIC works... yes
checking if x86_64-pc-linux-gnu-gcc static flag -static works... yes
checking if x86_64-pc-linux-gnu-gcc supports -c -o file.o... yes
checking if x86_64-pc-linux-gnu-gcc supports -c -o file.o... (cached) yes
checking whether the x86_64-pc-linux-gnu-gcc linker (/usr/x86_64-pc-linux-gnu/bin/ld -m elf_x86_64) supports shared libraries... yes
checking whether -lc should be explicitly linked in... no
checking dynamic linker characteristics... GNU/Linux ld.so
checking how to hardcode library paths into programs... immediate
checking whether stripping libraries is possible... yes
checking if libtool supports shared libraries... yes
checking whether to build shared libraries... yes
checking whether to build static libraries... no
checking for gperf... gperf
checking for antlr3... antlr3
checking sys/wait.h usability... yes
checking sys/wait.h presence... yes
checking for sys/wait.h... yes
checking sys/param.h usability... yes
checking sys/param.h presence... yes
checking for sys/param.h... yes
checking sys/select.h usability... yes
checking sys/select.h presence... yes
checking for sys/select.h... yes
checking dirent.h usability... yes
checking dirent.h presence... yes
checking for dirent.h... yes
checking regex.h usability... yes
checking regex.h presence... yes
checking for regex.h... yes
checking for posix_fadvise... yes
checking for strptime... yes
checking for strtok_r... yes
checking for timegm... yes
checking for euidaccess... yes
checking for special C compiler options needed for large files... no
checking for _FILE_OFFSET_BITS value needed for large files... no
checking for off_t... yes
checking for ld used by x86_64-pc-linux-gnu-gcc... /usr/x86_64-pc-linux-gnu/bin/ld -m elf_x86_64
checking if the linker (/usr/x86_64-pc-linux-gnu/bin/ld -m elf_x86_64) is GNU ld... yes
checking for shared library run path origin... done
checking for iconv... yes
checking for working iconv... yes
checking for iconv declaration... 
         extern size_t iconv (iconv_t cd, char * *inbuf, size_t *inbytesleft, char * *outbuf, size_t *outbytesleft);
checking for libunistring... yes
checking how to link with libunistring... -lunistring
checking for x86_64-pc-linux-gnu-pkg-config... /usr/bin/x86_64-pc-linux-gnu-pkg-config
checking pkg-config is at least version 0.9.0... yes
checking for ZLIB... yes
checking for CONFUSE... yes
checking for AVAHI... yes
checking for SQLITE3... yes
checking for sqlite3_unlock_notify in -lsqlite3... yes
checking if SQLite3 was built with threadsafe operations support... yes
checking for LIBAV... yes
checking for avcodec_find_best_pix_fmt_of_list in -lavcodec... yes
checking for MINIXML... yes
checking for LIBEVENT... yes
checking antlr3.h usability... yes
checking antlr3.h presence... yes
checking for antlr3.h... yes
checking for antlr3BaseRecognizerNew in -lantlr3c... yes
checking for antlr3NewAsciiStringInPlaceStream in -lantlr3c... no
checking for x86_64-pc-linux-gnu-libgcrypt-config... /usr/bin/x86_64-pc-linux-gnu-libgcrypt-config
checking for LIBGCRYPT - version >= 1.2.0... yes (1.5.4)
checking LIBGCRYPT API version... okay
checking for x86_64-pc-linux-gnu-gpg-error-config... /usr/bin/x86_64-pc-linux-gnu-gpg-error-config
checking for GPG Error - version >= 1.6... yes (1.13)
checking for FLAC... no
configure: error: Package requirements ( flac ) were not met:

No package 'flac' found

Consider adjusting the PKG_CONFIG_PATH environment variable if you
installed software in a non-standard prefix.

Alternatively, you may set the environment variables FLAC_CFLAGS
and FLAC_LIBS to avoid the need to call pkg-config.
See the pkg-config man page for more details.

!!! Please attach the following file when seeking support:
!!! /var/tmp/portage/media-sound/forked-daapd-23.2/work/forked-daapd-23.2/config.log
 * ERROR: media-sound/forked-daapd-23.2::alunduil failed (configure phase):
 *   econf failed
 * 
 * Call stack:
 *               ebuild.sh, line   93:  Called src_configure
 *             environment, line 3019:  Called econf '--with-alsa' '--disable-flac' '--disable-musepack' '--disable-itunes' '--disable-lastfm' '--disable-mpd'
 *        phase-helpers.sh, line  662:  Called __helpers_die 'econf failed'
 *   isolated-functions.sh, line  117:  Called die
 * The specific snippet of code:
 *              die "$@"
 * 
 * If you need support, post the output of `emerge --info '=media-sound/forked-daapd-23.2::alunduil'`,
 * the complete build log and the output of `emerge -pqv '=media-sound/forked-daapd-23.2::alunduil'`.
 * The complete build log is located at '/tmp/etest.logs.d/media-sound:forked-daapd-23.2:20150720-001435.log'.
 * For convenience, a symlink to the build log is located at '/var/tmp/portage/media-sound/forked-daapd-23.2/temp/build.log'.
 * The ebuild environment file is located at '/var/tmp/portage/media-sound/forked-daapd-23.2/temp/environment'.
 * Working directory: '/var/tmp/portage/media-sound/forked-daapd-23.2/work/forked-daapd-23.2'
 * S: '/var/tmp/portage/media-sound/forked-daapd-23.2/work/forked-daapd-23.2'

 * IMPORTANT: 15 news items need reading for repository 'gentoo'.
 * Use eselect news read to view new items.



------------------------------------------------------------------------
3 tests ran in 38684.379053 seconds
3 tests FAILED
```

### PR DESCRIPTION
**Pull request by erikmack**
Resolves #18 in alunduil overlay

### CODE REVIEW

File: media-sound/forked-daapd/forked-daapd-23.2.ebuild [hunk 1]
```diff
@@ -49,6 +49,9 @@ pkg_setup() {
 }
 
 src_prepare() {
+	# Required until upstream accepts https://github.com/ejurgensen/forked-daapd/pull/179
```
Comment (alunduil) [inline media-sound/forked-daapd/forked-daapd-23.2.ebuild]: Comment isn't necessary as I usually drop all patches on a bump (my style).  Glad to see you pushed the patch upstream as well as dropping it on the ebuild.

File: media-sound/forked-daapd/forked-daapd-23.2.ebuild [hunk 2]
```diff
@@ -70,7 +73,7 @@ src_install() {
 
 	# dodir by itself fails in the likely case of /srv/music having a
 	# volume mounted already.
-	test -d /srv/music || dodir /srv/music
+	[ -d /srv/music ] || dodir /srv/music
```
Comment (alunduil) [inline media-sound/forked-daapd/forked-daapd-23.2.ebuild]: Does test not work?  Why did you change this back?

Comment (erikmack) [inline media-sound/forked-daapd/forked-daapd-23.2.ebuild]: Sorry, the reversion was an artifact of my somehow not having it in my branch originally.  I've pushed the test change in another (non-squashed) commit.

**Comment by alunduil** [comment]
No worries.  Was just curious.  I'll merge this and add it to the tree when I have time to run my battery of testing against it.  Should be by Sunday.

**Comment by erikmack** [comment]
Thanks.  FWIW, I've got nine green etest dots so far, and that's after about 16 hours :+1:

**Comment by alunduil** [comment]
Yikes, might have to wait for the weekend then.  I'll be sure to start it Friday after work so it's ready by work time on Saturday.
